easybuild/tools/filetools.py

```diff
@@ -700,8 +700,15 @@ def adjust_permissions(name, permissionBits, add=True, onlyfiles=False, onlydirs
         for root, dirs, files in os.walk(name):
             paths = []
             if not onlydirs:
-                paths += files
+                if not skip_symlinks:
+                    paths += files
+                else:
+                    for path in files:
+                        fname = os.path.join(root, path)
+                        if not os.path.islink(fname):
```

Comment:
Right. This is a leftover from an intermediate version. Initially I appended `fname` rather than `path`, so it was actually used twice. But that was plain wrong...